packages/core/src/components/ModalNew/Modal/Modal.module.scss

```diff
@@ -19,6 +23,10 @@
   border-radius: var(--border-radius-big);
   box-shadow: var(--box-shadow-large);
 
+  &.withHeaderAction {
```

Comment:
What is it for?

Reply:
for basic layout (and other custom layouts by user) to know that the top actions width is larger